### PR TITLE
Update set-up-phone-system-voicemail.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7840
Added note

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-up-phone-system-voicemail.md [hunk 1]
```diff
@@ -70,6 +71,10 @@ To set up protected voicemail, do the following:
     > - [Configuring and managing templates for Azure Information Protection](/information-protection/deploy-use/configure-policy-templates)
     > - [Do Not Forward option for emails](/information-protection/deploy-use/configure-usage-rights#do-not-forward-option-for-emails)
 
+    > [!NOTE]
+    > For users, enterprises, and organizations that want the Voicemail form to appear, the following registry key needs to be set:
```
Comment (JohanFreelancer9) [inline Teams/set-up-phone-system-voicemail.md:75]:
```suggestion
    > You need to set the following registry key for users, enterprises, and organizations that want the Voicemail form to appear: 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @lindspea, kindly commit my suggested change if you agree with it. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**